src/test/java/org/kainos/ea/service/JobServiceUnitTest.java

```diff
@@ -64,4 +75,33 @@ void getAllJobs_shouldThrowFailedToGetJobsException_whenDatabaseConnectorThrowsS
                 () -> jobService.getAllJobs());
     }
 
+    @Test
+    void createJobRole_shouldReturnNewJobId_whenValidInputIsProvided() throws SQLException, ProjectException {
+
+        int expectedID = 1;
+
+        Mockito.when(databaseConnector.getConnection()).thenReturn(connection);
+        Mockito.when(jobDao.createJobRole(jobRequest, connection)).thenReturn(expectedID);
+
+        int result = jobService.createJobRole(jobRequest);
+
+        assertEquals(result, expectedID);
+    }
+
+
+
+    @Test
+    void createJobRole_shouldThrowProjectException_whenValidationFails() throws SQLException, ProjectException {
```

Comment:
this test isn't checking if validation fails. you should be mocking your validator and checking each possible scenario